front-end-components/src/contexts/hooks.tsx

```diff
@@ -39,3 +41,205 @@ export const useCentralServicesBreakdownContext = (
 
   return centralServicesBreakdownContext;
 };
+
+export const useCostCodeMapContext = (): CostCodeMapContextValue => {
+  const costCodeMap = useContext(CostCodeMapContext);
+
+  const getCostCodes = (category: string): string[] => {
+    if (!costCodeMap) {
+      return [];
+    }
+    switch (category) {
```

Comment:
Maybe add a comment here to advise that this mapping knowingly duplicates what is done in Web